src/cls/2pc_queue/cls_2pc_queue_types.h

```diff
@@ -10,6 +10,7 @@ struct cls_2pc_reservation
   inline static const id_t NO_ID{0};
   uint64_t size;        // how many entries are reserved
   ceph::coarse_real_time timestamp;  // when the reservation was done (used for cleaning stale reservations)
+  uint32_t entries;
```

Comment:
you can probably add `entries` to ctor and pass it directly in the `emplace` call

Reply:
Ack!